varcode/coding_effect.py

```diff
@@ -22,17 +22,54 @@
     Substitution,
     ComplexSubstitution,
     PrematureStop,
+    AlternateStartCodon,
     StartLoss,
     StopLoss,
     FrameShift,
     FrameShiftTruncation,
-    IncompleteTranscript,
     ThreePrimeUTR,
 )
 from .mutate import substitute, insert_after
 from .string_helpers import trim_shared_flanking_strings
 
-from Bio.Seq import Seq, CodonTable
+from Bio.Seq import Seq
+from Bio.Data import CodonTable
+
+START_CODONS = set(CodonTable.standard_dna_table.start_codons)
+STOP_CODONS = set(CodonTable.standard_dna_table.stop_codons)
+
+def translate(cds_seq):
+    """
+    Parameters
```

Comment:
a one sentence description of the logic this function follows might be helpful. This is probably second nature to you, but from reading this description it's not immediately clear to me what exactly `cds_seq` is -- e.g. does it include the UTR's?